src/app/admin/borrowers/borrower.ts

```diff
@@ -0,0 +1,7 @@
+export class Borrower {
+  constructor(
```

Comment:
Since you repeatedly call this constructor with "empty data" (0 and three empty strings), I suggest setting default values for the arguments (looking up the TypeScript syntax for that as needed), so other code can simply call `Borrower()`.

Reply:
Yes you are correct

Reply:
You are right